src/utils/getBoundsFromGeoServer.js

```diff
@@ -0,0 +1,32 @@
+import X2JS from 'x2js';
+
+export const getBoundsFromWmsLayer = (wmsSlug, wmsUrl, value, valueChanged) => {
+    // Use /proxy/ to avoid the request is blocked by CORS policy
+    // Note: in production, it will probably be working without the prefix /proxy/
+    const proxyUrl = `/proxy/${wmsUrl}`;
```

Comment:
If that comment is true, then this needs to be changed (we should only commit to master what works in production). But I think that on production this is also needed.

Reply:
Yeah you are right, because the request is sent to another server outside of Lizard so it is probably needed on production as well. I will remove this comment.